Makefile

```diff
@@ -0,0 +1,29 @@
+##########------------------------------------------------------------##########
+##########- Build Latest Tkn binary ------------------------------------------##########
+##########------------------------------------------------------------##########
+
+STABLE_BUILD_URL := https://github.com/tektoncd/cli/releases/download/v${TKN_VERSION}/tkn_${TKN_VERSION}_Linux_x86_64.tar.gz
+BUILD_PATH=build/tkn/v${TKN_VERSION}
+
+.PHONY: build-tkn-binary-Linux-x86-64
+build-tkn-binary-Linux-x86-64:
+ifndef TKN_VERSION
+	@echo TKN_VERSION not set
+	@exit 1
+endif
+	[[ -d "${BUILD_PATH}" ]] | mkdir -p ${BUILD_PATH}
```

Comment:
we should add `build` dir into `.gitignore`

Reply:
Yup, I missed it